app/routes/unified_search.py

```
@@ -0,0 +1,320 @@
+    value = value.replace("_", "\\_")
+    # Strip PostgREST filter delimiters that could inject conditions
+    value = value.replace(",", "")
+    value = value.replace(".", "")
+    return value
+
+
+# ============================================================================
+# Per-type search helpers
+# ============================================================================
+
+async def _search_chats(supabase, query_lower: str, raw_query: str, org_id, user_id, limit: int) -> List[SearchResultItem]:
```

Ruff (0.15.1)

[warning] Unused function argument: `query_lower`

(ARG001)